build/package/.goreleaser.yml

```diff
@@ -106,9 +106,15 @@ nfpms:
   dependencies:
     - mongodb-atlas-cli
     - mongodb-mongosh
-  recommends:
-    - podman
   meta: true
+  overrides:
+    deb:
+      dependencies:
+        - docker
+    rpm:
+      recommends:
+        - podman
+        - docker
```

Comment:
I don't think we can do docker the official package is docker-ce on linux right? (ex. https://docs.docker.com/engine/install/debian/)

Reply:
good catch, I've updated it